### ISSUE TITLE
Ngrok not starting

### ISSUE THREAD

**Issue by fellowmean5** [CacucoH/Seeker-Ngrok#1]
Hello, i m waiting 10 minutes,but ngrok not starting
![DBACAECC-0EC3-445F-8E32-08369B444E17](https://user-images.githubusercontent.com/73715753/110245157-bc755880-7f6a-11eb-80ac-36f433d2568a.png)

**Comment by CacucoH** [comment]
Hello! Thanks for your question, I think that it’s a bug with pyngrok, I will check and update program and check pyngrok lib

**Comment by fellowmean5** [comment]
Thanks сосисон)

**Comment by fellowmean5** [comment]
How mant time you need to fix this?
Thanks

**Comment by CacucoH** [comment]
Я думаю тут ошибка не в самой программе, а в библиотеке pyngrok. Я, конечно, ещё посмотрю, но хочу отметить, что ещё недели две назад все прекрасно работало)

**Comment by fellowmean5** [comment]
Слуушай, а как то версию постарее оригинального сикера достать можно?

**Comment by fellowmean5** [comment]
полкчилось запустить,но не приходят логи,что делать
![ываыва](https://user-images.githubusercontent.com/73715753/110294697-df4e4e00-7ff8-11eb-9ac1-4e84fc45c905.png)

**Comment by fellowmean5** [comment]
рабобрался,надо заменить файлы в папке templates / nearyou/php на файлы из ориг thewhiteh4t и дать им разрешение 777,тогда все топ будет

**Comment by CacucoH** [comment]
Спасибо за содействие, сейчас исправлю

**Comment by CacucoH** [comment]
Я исправил, переустанови репозиторий и запусти install.sh

**Comment by CacucoH** [comment]
Оно работает даже лучше чем оригинал: человеку даже не нужно нажимать на кнопку

**Comment by CacucoH** [comment]
Ещё в скором времени я планирую сделать русификацию

**Comment by fellowmean5** [comment]
Сейчас проверю...

**Comment by CacucoH** [comment]
Все работает? После того как iSH добавили в AppStore он начал работать ещё медленнее => для запуска pyngrok’a требуется уйма времени. Все испортили :(

**Comment by fellowmean5** [comment]
Да,я так и не дождался пока откроется.Слушай, у меня есть идея как сделать эту утилиту прям топ.Чтобы она записывала не только местоположение,но и делал фото жертвы. Можешь свои контакты оставить?
